hw/ip/otbn/doc/_index.md

```diff
@@ -1022,6 +1022,10 @@ In addition, it can be triggered by the [Life Cycle Controller]({{<relref "/hw/i
 
 A secure wipe of the internal state only is triggered automatically after reset and when OTBN [ends the software execution](#design-details-software-execution), either successfully, or unsuccessfully due to a [recoverable error](#design-details-recoverable-errors).
 
+If OTBN cannot complete a secure wipe of the internal state (e.g., due to failing to obtain the required randomness), it immediately becomes locked.
+In this case, OTBN must be reset and will then retry the secure wipe.
+The secure wipe after reset must succeed before OTBN can be used.
+
```

Comment:
@ctopal: Does this resolve this feedback you gave?
> On a more important note, as far as I understand, this countermeasure(not sure if it is) was not listed in the documentation at the first place. Can we add it to the documentation (reason below) please?
> 
> This change essentially drives the OTBN status to `LOCKED` but we say it's only fatal errors that can do that. Since every fatal error said to initiate secure wipe, this error does not qualify for a fatal error (according to the documentation anyways). So, some rewording of the documentation around that area is needed to at least solve that 
mismatch.

Reply:
I think it looks better yes, thank you! Although now I wonder if we need to change the RTL so that it would also go into `LOCKED` in the case of an incoming life cycle escalation or RMA request while doing internal secure wipe.

Reply:
> I wonder if we need to change the RTL so that it would also go into `LOCKED` in the case of an incoming life cycle escalation or RMA request while doing internal secure wipe.

The RTL already goes into `LOCKED` state in both cases, although not immediately as would be the case for a failed URND reseed. The rationale is that we want OTBN to complete the secure wipe whenever it can.

Reply:
Oh I see, that makes sense. Thanks for explaining!